src/applications/personalization/connected-accounts/components/ConnectedApps.jsx

```diff
@@ -30,13 +32,11 @@ export function ConnectedApps({ confirmDelete, propertyName, accounts }) {
         </table>
 
         <div className="feature">
-          <h3>Have questions about signing in to {propertyName}?</h3>
+          <h3>Have questions about connected accounts?</h3>
           <p>
-            Get answers to frequently asked questions about how to sign in,
-            common issues with verifying your identity, and your privacy and
-            security on {propertyName}.
+            Get answers to frequently asked questions about connected accounts
+            and how they work with your {propertyName} account.
```

Comment:
Looks like there’s an opportunity to hardcore everything to VA.gov in this file, too